### DISCUSSION TITLE
Status show down when it is up

### DISCUSSION THREAD

**Discussion by andikarekatias** [upptime/upptime#941] [Q&A]
**Describe the bug**
Website down but is up
Checking CI
Checking https://mediakepri.co.id/
Current status mediakepri down 2024-03-07T02:17:33.[27](https://github.com/andikarekatias/upptime/actions/runs/8182696265/job/22374435956#step:3:28)1Z
Result from test 403 0.243483
Result from test 403 0.006777
Result from test 403 0.021877
Skipping commit,  status is down

curl -qs -m 10 -o /dev/null -w "%{response_code}\n" https://mediakepri.co.id
200

**To Reproduce**
Checkout https://status.andikarekatias.com

**Expected behavior**
The expected result from the test is 200

**Additional context**
https://github.com/andikarekatias/upptime

**Comment by AnandChowdhary** [comment]
You can see from the commit that the website is responding with an HTTP 403. Could there be any reason why the service is responding with Forbidden when GitHub Actions tries to access it?

**Comment by andikarekatias** [comment]
i dunno thats why i ask in here. 
this is my robot.txt

User-Agent: *
Allow: /wp-content/uploads/
Disallow: /wp-content/plugins/
Disallow: /wp-admin/
Disallow: /readme.html
Disallow: /refer/
 
Sitemap: https://mediakepri.co.id/sitemap.xml
#Begin Attracta SEO Tools Sitemap. Do not remove
sitemap: http://cdn.attracta.com/sitemap/5634595.xml.gz
#End Attracta SEO Tools Sitemap. Do not remove

**Comment by andikarekatias** [comment]
i found it, managed challenge by cloudflare scurity
![image](https://github.com/upptime/upptime/assets/10099168/4722162b-4937-4e5b-9ceb-4bae7463f2eb)

so i create custome rule

![image](https://github.com/upptime/upptime/assets/10099168/890dc2a7-96b9-44ac-8fdb-6c2b9a12ca3d)

but still can't bypass cloudflare scurity, is my custom rule worng ?

this is my upptimerc.yml
- name: Mediakepri
    url: https://mediakepri.co.id
    headers:
      - "User-Agent: Andika Rekatias Uptime"

and my robot.txt
User-Agent: Andika Rekatias Uptime
Allow: /
